hapi-proto/src/main/proto/ScheduleSign.proto

```diff
@@ -27,6 +27,13 @@ option java_multiple_files = true;
 
 import "BasicTypes.proto";
 
+/*
+Provides signatures for a given Scheduled Transaction. The Scheduled Transaction executes if all of the required signatures are witnessed.
+
+INVALID_SCHEDULE_ID is returned if the specified ScheduleID does not exist, already executed or expired
```

Comment:
"INVALID_SCHEDULE_ID is returned if the specified ScheduleID does not exist, **has already been** executed or expired"